File: messaging/v1/service/create-service-create/create-service-create.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Messaging.V1;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var service = ServiceResource.Create(
            statusCallback: new Uri("http://requestb.in/1234abcd"),
            friendlyName: "My First Service"
        );

        Console.WriteLine(service.Sid);
    }
}

File: messaging/v1/service/create-service-create/create-service-create.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$service = $twilio->messaging->v1->services
                                 ->create("My First Service", // friendlyName
                                          array(
                                              "statusCallback" => "http://requestb.in/1234abcd"
                                          )
                                 );

print($service->sid);